src/snapred/backend/service/SousChef.py

```diff
@@ -121,20 +122,27 @@ def prepPeakIngredients(self, ingredients: FarmFreshIngredients) -> PeakIngredie
             peakIntensityThreshold=ingredients.peakIntensityThreshold,
         )
 
-    def prepDetectorPeaks(self, ingredients: FarmFreshIngredients) -> List[GroupPeakList]:
+    def prepDetectorPeaks(self, ingredients: FarmFreshIngredients, purge_peaks=True) -> List[GroupPeakList]:
```

Comment:
just a nitpick, but can you update purge_peaks to be camel case?

Reply:
Can do